Jenkinsfile

```diff
@@ -1,22 +1,31 @@
-// Setup milestone to stop previous build from running when a new one is launched
-// The result would be:
-//  Build 1 runs and creates milestone 1
-//  While build 1 is running, suppose build 2 fires. It has milestone 1 and milestone 2. It passes 1, which causes build #1 to abort
+@Library('jenkins-pipeline-shared-libraries')_
 
-def buildNumber = env.BUILD_NUMBER as int
-if (buildNumber > 1) milestone(buildNumber - 1)
-milestone(buildNumber)
+def changeAuthor = env.ghprbPullAuthorLogin ?: CHANGE_AUTHOR
+def changeBranch = env.ghprbSourceBranch ?: CHANGE_BRANCH
+def changeTarget = env.ghprbTargetBranch ?: CHANGE_TARGET
 
 pipeline {
-    agent { label 'operator-slave'}
+    agent { label 'kogito-operator-slave && !master'}
     options {
         buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')
         timeout(time: 90, unit: 'MINUTES')
     }
+    tools {
+            jdk 'kie-jdk11'
+        }
     environment {
+        TEMP_TAG="""pr-${sh(
+                returnStdout: true,
+                script: 'echo \${GIT_COMMIT} | cut -c1-7'
+            ).trim()}"""
+        JAVA_HOME = "${GRAALVM_HOME}"
+
         WORKING_DIR = "/home/jenkins/go/src/github.com/kiegroup/kogito-cloud-operator/"
```

Comment:
I don't think we need to be in a specific `WORKING_DIR` anymore

Reply:
okay, I'm not sure about this. I'll remove this and see if it works =D